backend/app/context/manager.py

```diff
@@ -26,7 +28,7 @@ def state_context(epoch_state: EpochState, is_simulated: bool = False) -> Contex
     return build_context(epoch_num, epoch_state, is_simulated)
 
 
-@cache.memoize(timeout=600)
+@cache.memoize(timeout=60)
```

Comment:
decreased, because I don't want to cache snapshot state for 10m. I believe it's gonna change when we introduce a producer for context, let's rethink it then